configs/processors/github.yaml

```
@@ -0,0 +1,91 @@
+    python: [function_definition, class_definition]
+    javascript: [function_declaration, class_declaration, arrow_function]
+    typescript: [function_declaration, class_declaration, arrow_function]
+    java: [method_declaration, class_declaration]
+    go: [function_declaration, type_declaration]
+    rust: [function_item, impl_item]
+    cpp: [function_definition, class_specifier]
+
+logging_level: INFO
+metrics_enabled: true
+
+# Vector index configuration  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)